packages/blockchain-wallet-v4-frontend/src/modals/OrderMyCard/FailedCreationStep/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+
+import { Image } from 'blockchain-info-components'
+import FlyoutContent from 'components/Flyout/Content'
+import { FlyoutHeader } from 'components/Flyout/Layout'
+
+import { ResultSubTitleWrapper, ResultTitleWrapper, ResultWrapper, StyledButton } from '../model'
+
+type Props = {
+  handleClose: () => void
+  handleRetry: () => void
+}
+
+const FailedCreationStep = ({ handleClose, handleRetry }: Props) => (
+  <>
```

Comment:
Could you import `FlyoutContainer` and use that here? Will just be a bit cleaner and future proof this modal if we ever change flyouts in the future.

Reply:
Thanks, I added that and I remove an unnecessary styling